content/integrations/entrust-keycontrol.md

```diff
@@ -0,0 +1,289 @@
+
+The identity is not a secret. It can be shared. Any peer
+needs this identity in order to verify your API key.
+
+The identity can be computed again via:
+
+    kes identity of kes:v1:AOlQZJRMYZeOioGk0ubYBMSFt1w6Hh1QZl3zG4PQxK/g
+```
+
+### Configure the KES Server
+
+Create the KES server configuration file: `config.yml`.
+Make sure that the identity in the policy section matches your API key identity.
+
+```yaml {.copy}
+address: 0.0.0.0:7373 # Listen on all network interfaces on port 7373
+
+admin:
+  identity: disabled  # We disable the admin identity since we don't need it in this guide 
+   
+tls:
+  key: private.key    # The KES server TLS private key
+  cert: public.crt    # The KES server TLS certificate
+   
+policy:
+  my-app: 
+    allow:
+    - /v1/key/create/my-key*
+    - /v1/key/generate/my-key*
+    - /v1/key/decrypt/my-key*
+    identities:
+    - eb559798a2fbcc3efbf036bed11108116e63f293324abdfe7574249ef5e56b36 # Use the identity of your API key
+   
+keystore:
+     entrust:
+       keycontrol:
+         endpoint: "https://keycontrol.my-org.com"    # Use your KeyControl instance endpoint.
+         vault_id: ""                                 # The Vault ID - e.g: "e30497c1-bff7-4e81-beb7-fb35c4b7410c".
+         box_id:   ""                                 # The Box name or ID - e.g: "tenant-1".
+         credentials:
+           username: ""                               # KeyControl username - e.g: "kes-tenant-1@my-org.com".
+           password: ""                               # Password of KeyControl user
+```
+
+### Start the KES Server
+
+Use the yaml file you created to start a KES server instance.
+
+```sh {.copy}
+kes server --config config.yml --auth off
+```
+
+{{< admonition title="Linux Swap Protection" type="tip" >}}
+
+In Linux environments, KES can use the [`mlock`](http://man7.org/linux/man-pages/man2/mlock.2.html) syscall to prevent the OS from writing in-memory data to disk (swapping). 
+This prevents leaking sensitive data.
+   
+Use the following command to allow KES to use the mlock syscall without running with `root` privileges:
+
+```sh {.copy}
+sudo setcap cap_ipc_lock=+ep $(readlink -f $(which kes))
+```
+
+Start a KES server instance with memory protection:
+   
+```sh {.copy}
+kes server --config config.yml --auth off --mlock
+```
+{{< /admonition >}}
+
+### KES CLI Access
+
+
+1. Set `KES_SERVER` Endpoint
+
+   The following environment variable specifies the server the KES CLI should talk to:
+
+   ```sh {.copy}
+   $ export KES_SERVER=https://127.0.0.1:7373
+   ```
+
+2. Define the Client Credentials
+
+   The following environment variables set the access credentials the client uses to talk to a KES server:
+   
+   ```sh {.copy}
+   $ export KES_CLIENT_CERT=client.crt
+   ```
+
+   ```sh {.copy}
+   $ export KES_CLIENT_KEY=client.key
+   ```
+
+3. Test the Configuration
+   
+   Perform any API operation allowed by the policy we assigned above. 
```

Comment:
Maybe? 
```suggestion
   Perform any API operation allowed by the policy assigned above. 
```
Perhaps `the policy defined in the KES server configuration file` even

Reply:
Taking the alternative you suggested.